src/test/java/com/microsoft/azure/kusto/kafka/connect/sink/it/KustoSinkIT.java

```diff
@@ -0,0 +1,311 @@
+    private static final KafkaContainer kafkaContainer = new KafkaContainer(DockerImageName.parse("confluentinc/cp-kafka:" + confluentVersion))
+            .withNetwork(network);
+    private static final ProxyContainer proxyContainer = new ProxyContainer().withNetwork(network);
+
+    private static final SchemaRegistryContainer schemaRegistryContainer = new SchemaRegistryContainer(confluentVersion).withKafka(kafkaContainer)
+            .withNetwork(network).dependsOn(kafkaContainer);
+    private static final List<String> testFormats = List.of("json", "avro", "csv"); // List.of("json", "avro", "csv", "raw"); // Raw for XML
+
+    private static ITCoordinates coordinates;
+    private static final KustoKafkaConnectContainer connectContainer = new KustoKafkaConnectContainer(confluentVersion)
+            .withNetwork(network)
+            .withKafka(kafkaContainer)
+            .dependsOn(kafkaContainer, proxyContainer, schemaRegistryContainer);
+
+    private static Client engineClient = null;
+
+    @BeforeAll
+    public static void startContainers() throws Exception {
+        coordinates = getConnectorProperties();
+        if (coordinates.isValidConfig()) {
+            ConnectionStringBuilder engineCsb = ConnectionStringBuilder.createWithAadApplicationCredentials(
+                    String.format("https://%s.kusto.windows.net/", coordinates.cluster),
+                    coordinates.appId, coordinates.appKey, coordinates.authority);
+            engineClient = ClientFactory.createClient(engineCsb);
+            log.info("Creating tables in Kusto");
+            createTables();
+            // Mount the libs
+            String mountPath = String.format(
+                    "target/components/packages/microsoftcorporation-kafka-sink-azure-kusto-%s/microsoftcorporation-kafka-sink-azure-kusto-%s/lib",
+                    Version.getVersion(), Version.getVersion());
+            log.info("Creating connector jar with version {} and mounting it from {},", Version.getVersion(), mountPath);
+            connectContainer.withFileSystemBind(mountPath, "/kafka/connect/kafka-sink-azure-kusto");
+            // createConnectorJar();
+            Startables.deepStart(Stream.of(kafkaContainer, schemaRegistryContainer, proxyContainer, connectContainer)).join();
+            log.info("Started containers , copying scripts to container and executing them");
+            connectContainer.withCopyToContainer(MountableFile.forClasspathResource("download-libs.sh", 744), // rwx--r--r--
+                    "/kafka/connect/kafka-sink-azure-kusto/download-libs.sh").execInContainer("sh", "/kafka/connect/kafka-sink-azure-kusto/download-libs.sh");
+            log.debug(connectContainer.getLogs());
+        } else {
+            log.info("Skipping test due to missing configuration");
+        }
+    }
+
+    private static void createTables() throws Exception {
+        URL kqlResource = KustoSinkIT.class.getClassLoader().getResource("it-table-setup.kql");
```

Comment:
The command .alter policy ingestionbatching 
alters the engine policy
batching policy is used by the DM , to refresh DM cache invoke 
    s""".refresh database '$databaseName' table '$tableName' cache ingestionbatchingpolicy"""
on the dm